docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -129,9 +129,7 @@ by the coordinator can be found in the ``dynamicFiltersStats`` structure.
     "dynamicFiltersStats" : {
           "dynamicFilterDomainStats" : [ {
             "dynamicFilterId" : "df_370",
-            "simplifiedDomain" : "[ [[2451546, 2451905]] ]",
-            "rangeCount" : 3,
-            "discreteValuesCount" : 0,
+            "simplifiedDomain" : "[ SortedRangeSet[type=bigint, ranges=3, {[2451546], ..., [2451905]}] ]",
```

Comment:
i feel this should be formatted one item per line for readability

Reply:
this is a copy-paste from actual query's queryinfo json